modules/components/pages/processors/cache.adoc

```diff
@@ -225,6 +226,10 @@ Retrieve the contents of a cached key and replace the original message payload
 with the result. If the key does not exist the action fails with an error, which
 can be detected with xref:configuration:error_handling.adoc[processor error handling].
 
+=== `exists`
+
+Check whether a specific key is already cached and replace the original message payload with `true` if the key exists in the cache, or `false` if it doesn't.
```

Comment:
```suggestion
Check if a specific key is in the cache and set the message payload to `true` if it exists, or `false` if it doesn’t.
```

Suggest this simplification.

Reply:
Updated but kept consistent with other descriptions.